src/Symfony/Component/Console/Input/Input.php

```diff
@@ -171,4 +191,62 @@ public function getStream()
     {
         return $this->stream;
     }
+
+    /**
+     * Returns a stringified representation of the options passed to the command.
+     * The options must NOT be escaped as otherwise passing them to a `Process` would result in them being escaped twice.
+     *
+     * @param string[] $optionNames Names of the options returned. If null, all options are returned. Requested options
+     *                              that either do not exist or were not passed (even if the option has a default value)
+     *                              will not be part of the method output.
+     *
+     * @return list<string>
+     */
+    public function unparse(?array $optionNames = null): array
```

Comment:
According to how you would use this method [in `webmozarts/console-parallelization`](https://github.com/webmozarts/console-parallelization/pull/328/files#diff-420c522b1de9b02d1f3c4163aaaabea35b6c12332f6c888738dff3cfc1226001R75-R76), you have a list of option names to exclude. This parameter should be `$excludedOptions`.

Reply:
In `WebmozartsConsoleParallelization` we know the options we do NOT want, hence to get the ones we need to forward we need to juggle with the command definition. It is a specific use case for that feature, but not the only one, for instance you could want those values for specific options.

That's why I thought it would be better in Symfony to have a more re-usable piece of code (which is also simpler) even thought that means a tiny bit more work on our side.